test/tests/chain_id.rs

```diff
@@ -0,0 +1,32 @@
+use rpc_test::test_config::TestConfig;
+use starknet::providers::{
```

Comment:
Import starknet_providers instead, we'll follow deoxys architecture (you can inspire from estimate fee)